skimage/feature/peak.py

```diff
@@ -159,3 +159,73 @@ def peak_local_max(image, min_distance=10, threshold_abs=0, threshold_rel=0.1,
         nd_indices = tuple(coordinates.T)
         out[nd_indices] = True
         return out
+
+
+def get_scale_local_maximas(cube_coordinates, laplacian_cube):
```

Comment:
I made some research and decided to vectorize this function instead.
I also rewrote this function in Cython, but the vectorized version works faster.
This is the related SO quetion:
http://stackoverflow.com/questions/29155745/speed-up-function-using-cython
All credits go to the author of that answer that vectorized this function.